### ISSUE TITLE
Real world measurements

### ISSUE THREAD

**Issue by LibrEars** [makehumancommunity/mpfb2#192]
Implement modeling sliders for body measurements in metrical units (they are available in the 0.0 to 1.0 range) similar to the [MH implementation](https://static.makehumancommunity.org/makehuman/docs/gender_random_measure_and_custom.html)

Related link: [ DIN 33402-2:2020-12 Ergonomics - Human body dimensions](https://www.dinmedia.de/en/standard/din-33402-2/324499732)

**Comment by joepal1976** [comment]
This is a work in progress. There are some initial work done in the "blender4_measure" branch: https://github.com/makehumancommunity/mpfb2/tree/blender4_measure

The related issue is #168 

@maryam-fatima might have some additional insight on this.

**Comment by YuYaDev** [comment]
I’ve noticed that development in blender4_measure seems to have stalled a bit. Could you clarify whether the current implementation is functional for practical use? Specifically, I’m wondering if there’s at least a minimal mapping of values (0–1) to real-world measurements (e.g., waist, hips, chest and height) for basic human modeling. If this isn’t yet implemented, would using MakeHuman as a workaround—exporting a model from there and importing it into Blender—be a viable solution? This is vital for my project. Thanks for any insights you can share!

**Comment by joepal1976** [comment]
No, the feature branch would need to be cleaned up and finished in order to be usable. There is no corresponding functionality in the released version of MPFB yet.

Going via MakeHuman is a viable path. I would recommend you "save" the character in MakeHuman rather than "export". MakeHuman Model files ("MHM") can be opened directly in MPFB. There is a button for this on the "new human" -> "from save file" panel in MPFB.
